stix_shifter/stix_translation/src/modules/elastic_ecs/json/from_stix_map.json

```diff
@@ -0,0 +1,54 @@
+{
+  "ipv4-addr": {
+    "fields": {
+      "value": ["source.ip", "destination.ip", "client.ip", "server.ip"]
+    }
+  },
+  "ipv6-addr": {
+    "fields": {
+      "value": ["source.ip", "destination.ip", "client.ip", "server.ip"]
+    }
+  },
+  "mac-addr": {
+    "fields": {
+      "value": ["source.mac", "destination.mac", "client.mac", "server.mac"]
+    }
+  },
+  "network-traffic": {
+    "fields": {
+      "src_port": ["source.port", "client.port"],
+      "dst_port": ["destination.port", "server.port"],
+      "protocols[*]": ["network.transport", "network.type", "network.protocol"],
+      "src_ref.value": ["source.ip", "client.ip"],
+      "dst_ref.value": ["destination.ip", "server.ip"]
+    }
+  },
+  "artifact": {
+    "fields": {
+      "payload_bin": ["event.original"]
+    }
+  },
+  "file": {
+    "fields": {
+      "name": ["process.name"]
+    }
+  },
+  "user-account":{
+    "fields": {
+      "user_id": ["user.name"]
+    }
+  },
+  "process" : {
+    "fields": {
+      "command_line": ["process.executable"],
+      "created": ["process.start"],
+      "pid": ["process.pid"],
+      "creator_user_ref.user_id": ["user.name"]
+    }
+  },
+  "url": {
+    "fields": {
+      "value": ["url.original"]
+    }
+  }
```

Comment:
You could also add a mapping for domain which would map to the ECS `url.domain` field.

Reply:
I am analysing the code to add suggested mapping.

Reply:
Updated code to include url.domain. Ref: https://github.com/IBM/stix-shifter/pull/131/commits/c6aee8a04a9c92b1648c734e5aaa325657fe74ef

Please review and let me know comments.